### PR TITLE
Remove bundlesize dependency

### PR DESCRIPTION
**Pull request by Krisztiaan**
While testing this repo with `pnpm` (it's about 30% faster, but probably not worth it for the cons), I came across a concerning time hog:

```
iltorb: Running install script, done in 22.2s
```

`iltorb` is in `brotli-size` is in `bundlesize`

The script runs for about half of the install time, but at first glance, it looks somewhat unused, so it may be safe to delete, to gain a massive `yarn install` boost (from 56s mean to 33s mean install time on MBP16 i9).

Alternative: [bundlesize beta channel](https://github.com/siddharthkp/bundlesize/pull/324) has this resolved by making brotli optional

### CODE REVIEW

**Comment by peterp** [comment]
This was added incase someone imported something from a auth provider that wasn't a  type. Long term we do not want to be responsible for the various auth providers, so this is a good chance.

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+link-workspace-packages=true
```
Comment (peterp) [inline .npmrc:1]: Accidental file?

**Review by peterp — CHANGES_REQUESTED** [review]

File: .npmrc (hunk 1 above)
Comment (Krisztiaan) [reply .npmrc:1]: Yeah, remaining from yarn+lerna -> pnpm experiment

**Review by peterp — APPROVED** [review]
Thanks!

**Pull request merged**